modules/api/docs/LorisRESTAPI.md

```diff
@@ -787,82 +787,184 @@ The JSON object is of the form:
-    [
+    "DicomTars" : [
         {
             "Tarname" : "DCM_yyyy-mm-dd_ImagingUpload-hh-mm-abc123.tar",
-            "SeriesInfo" :
-                [{
+            "SeriesInfo" : [
+                {
                     "SeriesDescription" : "MPRAGE_ipat2",
-                    "SeriesNumber" : "2",
+                    "SeriesNumber" : 2,
                     "EchoTime" : "2.98",
                     "RepetitionTime" : "2300",
                     "InversionTime" : "900",
                     "SliceThickness" : "1",
                     "Modality" : "MR",
                     "SeriesUID" : "1.2.3.4.1107",
-                    },
-                    {
+                },
+                {
                     "SeriesDescription" : "BOLD Resting State",
-                    "SeriesNumber" : "5",
+                    "SeriesNumber" : 5,
                     "EchoTime" : "30",
                     "RepetitionTime" : "2100",
                     "InversionTime" : NULL,
                     "SliceThickness" : "3.5",
                     "Modality" : "MR",
                     "SeriesUID" : "3.4.5.6.1507",
-                }]
+                }
+            ]
         },
         {
             "Tarname" : "DCM_yyyy-mm-dd_ImagingUpload-hh-mm-def456.tar",
-            "SeriesInfo" :
-                [{
-                "SeriesDescription" : "MPRAGE_ipat2",
-                "SeriesNumber" : "2",
-                "EchoTime" : "2.98",
-                "RepetitionTime" : "2300",
-                "InversionTime" : "900",
-                "SliceThickness" : "1",
-                "Modality" : "MR",
-                "SeriesUID" : "1.7.8.9.1296",
-                }]
+            "SeriesInfo" : [
+                {
+                  "SeriesDescription" : "MPRAGE_ipat2",
+                  "SeriesNumber" : 2,
+                  "EchoTime" : "2.98",
+                  "RepetitionTime" : "2300",
+                  "InversionTime" : "900",
+                  "SliceThickness" : "1",
+                  "Modality" : "MR",
+                  "SeriesUID" : "1.7.8.9.1296",
+                }
+            ]
         }
-    ]    
+    ]
 }
 ```
 
-The `Modality` header in the SeriesInfo is either `MR` or `PT` for MRI or PET 
+The `Modality` header in the SeriesInfo is either `MR` or `PT` for MRI or PET
 scans, respectively.
 
+```
+POST /candidates/$CandID/$VisitLabel/dicoms
+```
+
+The body of the POST request should be encoded as a form data and contain a
+field `File` with the imaging file, and `Json`, with the following keys and
+values:
+
+```js
+{
+    "CandID": $CandID,
+    "PSCID": $PSCID,
+    "VisitLabel": $VisitLabel,
+    "IsPhantom": boolean
+}
+```
+
+The JSON can optionally have a boolean `Overwrite` attribute that can be used
+to overwrite an existing file (`false` if not present).
+A successful request will be answered by a `303 See Other` response with its
+`Location` header pointing to the processes list of the new upload.
```

Comment:
Shouldn't it be pointing to the new upload?

If an API user wanted the process list instead of the file itself after uploading a file, they could just append "/processes".